src/components/pages/order/main/admin/adminPanel/editForm/EditForm.js

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from "react";
+import GlobalContext from "../../../../../../../context/GlobalContext";
+import styled from "styled-components";
+import AdminForm from "../../../../../../reusable-ui/AdminForm";
+import EmptyEdit from "./EmptyEdit";
+
+export default function EditForm() {
+  const { productSelected, setProductSelected, handleEdit, titleEditBoxRef } =
+    useContext(GlobalContext);
+
+  const handleChange = (e) => {
+    setProductSelected({
+      ...productSelected,
+      [e.target.name]: e.target.value,
+    });
+    handleEdit(productSelected, e);
+  };
+
+  if (!productSelected) {
+    return <EmptyEdit />;
+  }
+
+  return (
+    <EditFormStyled>
+      <AdminForm
+        product={productSelected}
+        onChange={handleChange}
+        titleEditBoxRef={titleEditBoxRef}
+      />
+      <p>
```

Comment:
```suggestion
      <span>
```

c'est du texte oui mais qui n'est pas dans un paragraphe.